platform-sdk/swirlds-logging/src/main/java/com/swirlds/logging/utils/ConfigUtils.java

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.logging.utils;
+
+import static com.swirlds.logging.utils.ConfigUtils.DataUnit.BYTE;
+import static com.swirlds.logging.utils.ConfigUtils.DataUnit.GIGA_BYTE;
+import static com.swirlds.logging.utils.ConfigUtils.DataUnit.KILO_BYTE;
+import static com.swirlds.logging.utils.ConfigUtils.DataUnit.MEGA_BYTE;
+import static com.swirlds.logging.utils.ConfigUtils.DataUnit.TERA_BYTE;
+
+import com.swirlds.config.api.Configuration;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.Objects;
+
+/**
+ * Ad-hoc configuration utilities.
```

Comment:
Maybe add the issue for moving this to config module and/or `@Deprecated(forRemoval = true)`

Reply:
added the issue number plus annotation.
Reference to the issue: https://github.com/hashgraph/hedera-services/issues/12310